app/services/slack_notification_service.rb

```diff
@@ -65,4 +64,25 @@ def delete_message(matches)
   def on_hold?(label)
     label.downcase == ON_HOLD.downcase
   end
+
+  def message
+    pull_request_url = extra_params[:pull_request][:html_url]
+    language = extra_params[:repository][:language]
+    "#{pull_request_url} #{language_emoji(language)}"
+  end
+  
+  def language_emoji(language)
+    language_emoji_hash.fetch language, "[#{language}]"
+  end
+  
+  def language_emoji_hash
```

Comment:
I think this should be a Constant and not a method.

Reply:
@matiasmansilla1989 updated